README.md

```diff
@@ -5,6 +5,27 @@
 ## Overview
 Library used to get and validate tokens for Tazama.
 
+## Installation
+
+A personal access token is required to install this repository. For more information read the following.
+https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#about-scopes-and-permissions-for-package-registries
+
+Thereafter you can run 
+  > npm install @frmscoe/auth-lib
```

Comment:
Should this still be @frmscoe?

Reply:
**@tazama-lf** does not currently have a published artifact for `auth-lib`. We should probably publish one before we change it

Reply:
Renamed.